Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -208,6 +208,9 @@ class Connection : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Connec
     public:
         virtual void didClose(Connection&) = 0;
         virtual void didReceiveInvalidMessage(Connection&, MessageName) = 0;
+#if USE(EXTENSIONKIT)
+        virtual bool requestRemoteProcessTermination() { return false; }
```

Comment:
Should be pure virtual, as mentioned in my earlier review.

Reply:
Fixed!